photos.css

```diff
@@ -76,16 +76,41 @@ p {
     background: white;
 }*/
 
+#bigdiv img {
+	width: 100%;
+}
+
 #bigdiv {
 	line-height: 170%;
 	-webkit-row-gap:      10px;
 	-webkit-column-count: 3;
 	-webkit-column-gap:   10px;
 	column-count:         3;
-	column-gap:           10px;  
+	column-gap:           10px;
 	row-gap:              10px;
 	padding: 10px;
+}
+
+/* tablet styles */
+@media only screen and (max-width : 800px) {
+	#bigdiv {
+		-webkit-row-gap:      10px;
+		-webkit-column-count: 2;
+		-webkit-column-gap:   10px;
+		column-count:         2;
+		column-gap:           10px;
+		row-gap:              10px;
+		padding: 0;
+	}
+
+}
 
+/* smartphone styles */
+@media only screen and (max-width : 440px) {
```

Comment:
and then when we get to 440px or below, apply these styles (change the layout to 1 column)